pkg/kubelet/dockertools/docker.go

```diff
@@ -534,19 +533,29 @@ var (
 	ErrContainerCannotRun = errors.New("Container cannot run")
 )
 
-func inspectContainer(client DockerInterface, dockerID, containerName, tPath string) (*api.ContainerStatus, error) {
+// Internal information kept for containers from inspection
+type containerStatusResult struct {
+	status api.ContainerStatus
+	podIP  string
```

Comment:
why is `inspectContainer` returning 'Pod'IP? Why not name it just ip or container ip?